packages/core/src/sass/globals.scss

```diff
@@ -47,7 +47,7 @@ $uikit-maxwidth: 42em; //Answer to life, the universe, and everything (keeps thi
 $uikit-colour-Hero: #115361 !global; //color variables (yes! the actual spelling doesn't have a u)
```

Comment:
My two cents, yes. This is why I "request changes" on the review. LMK where you both land. In the meantime, i'll lift the "Request Changes".